src/site/content/en/blog/stories-component/index.md

```diff
@@ -81,30 +137,49 @@ Parent with overscroll defines snap behavior
 }
 ```
 
-{% CompareCaption %}
-Children opt into being a snap target
-{% endCompareCaption %}
+```css/1-1
+.story {
+  grid-area: story;
+  background-size: cover;
+  …
+}
+```
 
-{% endCompare %}
-</div>
+This puts our HTML in control of the stacking order and also keeps all elements
+in flow. Notice how we didn't need to do anything with `absolute` positioning or `z-index` and
+we didn't need to box correct with `height: 100%` or `width: 100%`. The parent grid
+already defined the size of the story picture viewport, so none of these story components
+needed to be told to fill it!
 
-{% Aside 'key-term' %}
-Here's a great breakdown of [`scroll-snap-points` by Sarah Drasner](https://css-tricks.com/introducing-css-scroll-snap-points/) and [the CSS spec](https://www.w3.org/TR/css-scroll-snap-1/) for reference.
-{% endAside %}
+### CSS Scroll Snap Points {: #scroll-snap-points}
 
+The [CSS Scroll Snap Points spec](https://www.w3.org/TR/css-scroll-snap-1/) makes it
+a cinch to lock elements into the viewport on scroll. Before these CSS properties existed, 
+you had to use JavaScript, and it was… tricky, to say the least. Check out
+[Introducing CSS Scroll Snap Points](https://css-tricks.com/introducing-css-scroll-snap-points/)
+by Sarah Drasner for a great breakdown of how to use them.
```

Comment:
This link was previously at the bottom of the post but we want it at the top because it serves as a way for readers to get up to speed if they're not familiar with Snap Points